File: CPP_Notes/OOPS.py
##Encapsulation:

class Encapsulation:
    def __init__(self):
        self.__balance = 0

    def deposit_amount(self,amount):
        if(amount>0):
            self.__balance += amount    

    def get_balance(self):
        return self.__balance


if __name__ == "__main__":
    account = Encapsulation()
    account.get_balance()             

File: CPP_Notes/OOPS.cpp
#include<iostream>
using namespace std;

class Encapsulation{
private:
    int balance;

public:
    Encapsulation()
    {
        balance = 0; 
    }
    void depositBalance(int amount)
    {
        if(amount>0)
            balance += amount;
    }
    
    int GetBalance()
    {
        return balance;
    }
};

int main(){
    Encapsulation account;
    account.GetBalance();
}
